alibi_detect/saving/loading.py

```diff
@@ -492,15 +495,15 @@ def resolve_config(cfg: dict, config_dir: Optional[Path]) -> dict:
         elif isinstance(src, dict):
             backend = cfg.get('backend', 'tensorflow')
             if key[-1] in ('model', 'proj'):
-                obj = _load_model_config(src, backend)
+                obj = _load_model_config(src)
             elif key[-1] == 'embedding':
-                obj = _load_embedding_config(src, backend)
+                obj = _load_embedding_config(src)
             elif key[-1] == 'tokenizer':
                 obj = _load_tokenizer_config(src)
             elif key[-1] == 'optimizer':
                 obj = _load_optimizer_config(src, backend)
```

Comment:
Some of these functions still have a `backend` arg since we still want some of these objects' backends to be constrained by the detector `backend`. For example, it would not make sense to allow a pytorch optimizer to be loaded with a tensorflow backend detector.